RELEASE.md

```diff
@@ -69,6 +24,12 @@ The script makes use of several Travis environment variables to determine which
 1. In a release build, it looks at the release tag and builds only the stack with matching id.
 1. If none of these criteria is matched, it will include all stacks.
 
+### Lint
```

Comment:
Please pull latest from master and merge only ur changes on top of the latest contents